pages/windows/prompt.md

```diff
@@ -0,0 +1,24 @@
+# prompt
+
+> Change the default DOS style prompt in a command window.
+> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/prompt>.
+
+- Reset the prompt to default 'C:\>':
+
+`prompt`
```

Comment:
```suggestion
- Reset the prompt to the default `C:\>`:

`prompt`
```

Optional fix: `the default` may sound better.
Optional fix: backticks can be used instead of single quotes to be consistent with other pages.